### DISCUSSION TITLE
the editor show pandoc table in strange format

### DISCUSSION THREAD

**Discussion by braindevices** [KDE/ghostwriter#827] [Q&A]
just copied the table example direct from pandoc documentation:
```
-------------------------------------------------------------
 Centered   Default           Right Left
  Header    Aligned         Aligned Aligned
----------- ------- --------------- -------------------------
   First    row                12.0 Example of a row that
                                    spans multiple lines.

  Second    row                 5.0 Here's another one. Note
                                    the blank line between
                                    rows.
-------------------------------------------------------------

Table: Here's the caption. It, too, may span
multiple lines.
```

it says `ghostwriter 2.2.0` but I am pretty sure it is 2.1.6.

The preview result (pandoc style) is totally fine. However, in the editor area, it shows very strange format.

![image](https://user-images.githubusercontent.com/7507440/194759571-4c7b1034-c0cd-480f-aea2-fdc0df7f6cdc.png)

**Comment by braindevices** [comment]
normal user cannot create issue on invent.kde.org. I have to put it here.

**Comment by wereturtle** [comment]
1.  Please make an account on bugs.kde.org and report the issue there.
2. Were you using the default cmark-gfm processor, or Pandoc with ghostwriter?  (Pandoc must be installed manually for your Pandoc tables to work.  There is an option in the preview settings to choose your Markdown processor once you have installed it.)

**Reply by braindevices** [reply]
1. ok, so officially we now start to track with bugs.kde.org https://bugs.kde.org/show_bug.cgi?id=460526
2. I am using pandoc for preview, the preview works totally fine. The image I posted is in editor area not in preview area.

**Reply by braindevices** [reply]
personally I really hate KDE bug tracking system. Really like the github or gitlab issues.
